src/strands/experimental/bidi/models/openai.py

```
@@ -127,45 +125,35 @@ async def start(
             messages: Conversation history to initialize with.
             **kwargs: Additional configuration options.
         """
-        if self._active:
-            raise RuntimeError("Connection already active. Close the existing connection before creating a new one.")
+        if self._connection_id:
+            raise RuntimeError("call stop before starting again")
 
         logger.info("openai realtime connection starting")
 
-        try:
```

The remaining changes here are:
* Remove unused `self._active`.
* Prefix connection_id and websocket with _ to make private.
* Remove try/except. For all model providers, we are going to let errors propagate and handle logging centrally in the agent loop.